ERCS/_onchain-metadata.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: XXXX
```

Comment:
```suggestion
eip: 8048
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.